build/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -38,26 +44,25 @@ export default () => {
       const behaviorVariantName = file.basename
       const behaviorName = path.basename(dir)
 
-      let description
+      let descriptionText
+      let specificationText
       const fileContent = fs.readFileSync(file.path).toString()
       const blockComments = extract(fileContent).filter(comment => comment.type === 'BlockComment') // filtering only block comments
-      const emptyDescriptionText = 'Behavior file has no description.'
 
-      // getting object that describes '@description' part of the comment's text
+      // getting specification of the comment's text
```

Comment:
nit: actually, it is about `getting 'description' AND 'specification' from the comment's text